flower_classifier/datasets/oxford_flowers.py

```diff
@@ -178,6 +201,10 @@ def __getitem__(self, index):
     def __len__(self):
         return len(self.labels)
 
+    @property
+    def classes(self):
+        return NAMES
+
 
 class OxfordFlowersDataModule(pl.LightningDataModule):
     def __init__(self, data_dir=ROOT_DATA_DIR, batch_size=64):
```

Comment:
Should we break out a `@property` for `len_train` and `len_valid` in this DataModule too for consistency?